README.md

```diff
@@ -1,12 +1,17 @@
 # Developer Portal
-The [developer portal](http://developers.invoca.net) contains public documentation about our APIs. The documentation is hosted by [ReadTheDocs](http://readthedocs.org) (RTD). All the documentation is contained in this repo and is written in [reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText) markup language.
+
+The [developer portal](http://developers.invoca.net) contains public documentation about our APIs. The documentation is hosted by [ReadTheDocs](http://readthedocs.org) (RTD). All the documentation is contained in this repo and is written in [reStructuredText](https://en.wikipedia.org/wiki/ReStructuredText) markup language, and built using the [Sphinx Document Generator](http://www.sphinx-doc.org/en/stable/index.html). If you need to make any changes to conf.py please review the [Sphinx Dev Guide](http://www.sphinx-doc.org/en/stable/devguide.html)
 
 IMPORTANT things to note:
 
 1. Changes made to this repo are AUTOMATICALLY picked up by RTD and published to the site.
-2. You should no longer update the documentation that is present in the API controllers. This documentation will be removed shortly.
-3. A branch on git maps to a "version" on read-the-docs. To create a new version (branch) and have it displayed as default, you must go into read the docs admin and point "default" to the new branch. At that point, the old default will become an older version, so **do not** delete old branches of the form YYYY-MM-DD.
+2. The method of adding comments to API controllers is officially deprecated.
+3. A branch on git maps to a "version" on read-the-docs. To create a new version (branch) and have it displayed as default,
+ you must go into read the docs admin and point "default" to the new branch. At that point, 
+ the old default will become a legacy version that is still supported, so **do not** delete any old branches of the form YYYY-MM-DD.
 4. There is no master branch for this repo. If you are making a new version, you are responsible for setting the default branch of this repo to point to your new branch (version).
+5. Beware pitfalls. There are many. Dig into the shinx source code if need be. 
+6. Any Major changes involving a change to the toctree hierarchy  must be reviewed by MikeW, Spencer, or NickB.
```

Comment:
toctree is what they call that nav menu generated thing on the left side. It's a directive in Sphinx.